README.md

```diff
@@ -59,36 +59,51 @@ After configuration is complete, running the link sharing is as simple as:
+0. Upload your static site and other files to tardigrade using [Uplink](https://github.com/storj/storj/wiki/Uplink-CLI) 
+or [S3 gateway](https://documentation.tardigrade.io/api-reference/s3-gateway). Download the [Uplink Binary](https://github.com/storj/storj/wiki/Uplink-CLI).
+
+1. Share your READONLY objects via `uplink share --readonly --dns <hostname> sj://<bucket/prefix>`.
+
+This command will provide the info needed to create your 3 dns records.
+
+For example `uplink share --dns yourHostName sj://bucket/prefix` will output:
+
+```
+Type CNAME, Hostname yourHostName, Target link.tardigradeshare.io.
+------
+Type TXT, Hostname txt-yourHostName, Content storj-root:bucket/prefix
+------
+Type TXT, Hostname txt-yourHostName, Content storj-access:3WXG1qE
+```
+
+2. Create a CNAME record on your hostname using our linksharing common URL `link.tardigradeshare.io.` as the target name.
    
+    `Type CNAME, Hostname yourHostName, Target link.tardigradeshare.io.`
+    
     <img src="docs/images/cname.png" width="50%">
 
-3. Create 3 TXT records with the following info. You will need to add the entire string (including the prefix) to your records.
-   
-    a. `storj_grant-1` is the first part of your access grant.
+3. Create 2 TXT records, prepending `txt-` to your hostname.
+    
+    a. Root Path: the bucket, object prefix key, or individual object that you want your root domain to resolve to.
    
-    <img src="docs/images/grant1.png" width="50%">
+    `Type TXT, Hostname txt-yourHostName, Content storj-root:bucket/prefix`
     
-    b. `storj_grant-2` is the second part of your access grant. The two parts don't need to be exactly 1/2 of the entire access grant.
-     We require at least 2 strings because of txt record length restrictions. If your access grant is more than 450 characters long, you might need to split it into even more sections.
+    <img src="docs/images/root.png" width="50%">
     
-    <img src="docs/images/grant2.png" width="50%">
+    b. Access Key: the readonly and public access key to your root path.
     
-    c. `storj_root` is the path of the shared object. It may be the path to a bucket, a directory, or an individual object.
-   
-    <img src="docs/images/bucket.png" width="30%">
-    <img src="docs/images/dir.png" width="30%">
-    <img src="docs/images/obj.png" width="30%">
+    `Type TXT, Hostname txt-yourHostName, Content storj-access:3WXG1qE`
     
-4. That's it! You should be all set to share your files or directories with your custom domain.
+    <img src="docs/images/access.png" width="50%">
+
+4. You can check to make sure your dns records are ready with `dig @1.1.1.1 txt-<yourHostName>.<yourDomain> TXT`        
+    
+5. Without further action, your site will be served with http. You can secure your site by using a https proxy server such as [Cloudflare](https://www.cloudflare.com/)
+
+6. That's it! You should be all set to access your website e.g. `http://yourHostName.yourwebsite.com`
```

Comment:
Use either `example.com` or `website.test`. `yourwebsite.com` is a public website that does not belong to storj.